src/app/[locale]/join-us/__tests__/__snapshots__/page.test.tsx.snap

```diff
@@ -59,19 +59,19 @@ exports[`Support Us page > renders correctly 1`] = `
             <div
               class="text-rose-600 text-base font-bold leading-normal"
             >
-              Support Us
+              Join Us
             </div>
           </div>
           <div
             class="self-stretch text-center text-white text-[56px] lg:text-[80px] leading-[114.286%] md:leading-[120%] font-semibold"
           >
-            What we’re trying to achieve
+            Join us in ensuring the future of open source Java
```

Comment:
I'm not sure this phrasing is appropriate.  OpenJDK upstream is OSS (and always will be), I think we want to say something like "Join us in ensuring the future of world-class, open, enterprise-ready Java runtimes"

Reply:
I agree that this phrase requires some rethinking, as the implications just feel wrong. It is subtle, but there is good value in the Temurin project that we don't need to be writing anything that is open to misinterpretation or confusion.